modules/gapi/src/api/gmat.cpp

```diff
@@ -33,22 +33,47 @@ const cv::GOrigin& cv::GMat::priv() const
     return *m_priv;
 }
 
+template <typename T> std::vector<cv::GMatDesc> vec_descr_of(const std::vector<T> &vec)
+{
```

Comment:
+ please also make this function `static` or wrap with an anonymous `namespace`

Reply:
+1 for anonymous  `namespace`